### ISSUE TITLE
realization.load_status() can crash on input

### ISSUE THREAD

**Issue by berland** [equinor/fmu-ensemble#90]
Stack-trace:
```
The script 'ExternalErtScript' caused an error while running:
Traceback (most recent call last):
 File "/pr../wf_well_volumes.py", line 458, in <module>
   main()
 File "/projec.......wf_well_volumes.py", line 354, in main
   ens             = ensemble.ScratchEnsemble('ens', args.scratch_dir+'/realization-*')
 File "/project/res/lib/python2.7/site-packages/fmu/ensemble/ensemble.py", line 125, in __init__
   paths, realidxregexp, autodiscovery=autodiscovery
 File "/project/res/lib/python2.7/site-packages/fmu/ensemble/ensemble.py", line 228, in add_realizations
   realdir, realidxregexp=realidxregexp, autodiscovery=autodiscovery
 File "/project/res/lib/python2.7/site-packages/fmu/ensemble/realization.py", line 137, in __init__
   self.load_status()
 File "/project/res/lib/python2.7/site-packages/fmu/ensemble/realization.py", line 463, in load_status
   hms = list(map(int, jobrow["STARTTIME"].split(":")))
ValueError: invalid literal for int() with base 10: '1/Process'
```

**Comment by berland** [comment]
Two last lines of `STATUS` was:
```
ECLIPSE100_2018.2               : 10:40:56 .... 21:04:20  
08:17:06   EXIT: 1/Process exited with status code 1
```

**Comment by berland** [comment]
The appearance of this bug is probably related to an ERT upgrade, and amplifies the need to fix issue #12

**Comment by berland** [comment]
Bug credit: bkh
